pinot-common/src/main/java/com/linkedin/pinot/common/utils/FileUploadDownloadClient.java

```diff
@@ -235,9 +236,15 @@ private static String getErrorMessage(HttpUriRequest request, CloseableHttpRespo
       controllerVersion = response.getFirstHeader(CommonConstants.Controller.VERSION_HTTP_HEADER).getValue();
     }
     StatusLine statusLine = response.getStatusLine();
+    String message = null;
+    try {
+      message = EntityUtils.toString(response.getEntity());
+    } catch (Exception e) {
+      // ignore
+    }
     String errorMessage =
-        String.format("Got error status code: %d with reason: %s while sending request: %s", statusLine.getStatusCode(),
-            statusLine.getReasonPhrase(), request.getURI());
+        String.format("Got error status code: %d with reason: %s while sending request: %s\nMessage: %s", statusLine.getStatusCode(),
+            statusLine.getReasonPhrase(), request.getURI(), message);
```

Comment:
Is this the only instance that we missed adding the message or do we need to d a sweep and add it everywhere?